python_api/my_python_api/routers/users.py

```diff
@@ -12,32 +17,65 @@
 
 # Register endpoint
 @router.post("/register", status_code=status.HTTP_201_CREATED, response_model=schemas.UserRegisterOut)
-def register_user(newUser: schemas.UserRegister):
+def register_user(newUser: schemas.UserRegister, db: Session = Depends(get_db)):
     # Sends call to auth server
     address = f"{authServerURL}/register"
     response = requests.post(address, json=newUser.model_dump(), headers={"Content-Type": "application/json"})
```

Comment:
Not now but later on we might want to create a client class that its whole purpose is to make http call to the auth server. This way we dont have to do request.post etc every function that requires to communicate to the auth server. Not now though